sdk/python/testvectors/__main__.py

```diff
@@ -176,7 +176,7 @@ def create_block(self, test_name, original_descriptor):
 	@staticmethod
 	def create_manually(test_name, original_descriptor):
 		del test_name
-		return original_descriptor['object'], None
+		return original_descriptor['object'], str(original_descriptor['object'])
```

Comment:
do you need str here ?

Reply:
Probably not but let me check

Reply:
I left the str() or else it fails trying to convert to JSON with ``TypeError: Object of type DetachedCosignature is not JSON serializable``